bpfman/src/utils.rs

```diff
@@ -185,3 +185,25 @@ pub(crate) fn sled_insert(db_tree: &Tree, key: &str, value: &[u8]) -> Result<(),
         )
     })
 }
+
+// Helper function to get the error message from stderr
```

Comment:
Andrew:
Made multiple comments about logging errors and then bailing with basically the same error.

Andre:
I agree.  This is not ideal.  I was doing it because the error wasn't getting back to the calling program.  However, I did see that stderr was being returned, which contained the error message, so I implemented this function to pull the error out so it could be logged by the calling function.  I'm not sure if this is the best way to do this, but I'm open to other ideas.

Reply:
This looks good and is what I was thinking 👍